applications/luci-app-cloudflared/Makefile

```diff
@@ -0,0 +1,18 @@
+# SPDX-License-Identifier: GPL-3.0-only
+#
+# Copyright (C) 2024 Hilman Maulana <hilman0.0maulana@gmail.com>
+
+include $(TOPDIR)/rules.mk
+
+LUCI_TITLE:=LuCI for Cloudflared
+LUCI_DEPENDS:=+cloudflared
+LUCI_PKGARCH:=all
+LUCI_DESCRIPTION:=LuCI support for Cloudflared
+
+PKG_MAINTAINER:=<https://github.com/animegasan>
```

Comment:
The maintainer must be name and email like `Hilman Maulana <hilman0.0maulana@gmail.com>`